src/main/java/taxi/dao/jdbcimpl/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,133 @@
+        String query = "SELECT * FROM manufacturers "
+                + "WHERE manufacturer_id = ? "
+                + "AND deleted = FALSE;";
+
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            preparedStatement.setLong(1, id);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            if (resultSet.next()) {
+                Optional<Manufacturer> newManufacturer = Optional
+                        .of(createManufacturer(resultSet));
+                resultSet.close();
+                return newManufacturer;
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can not find manufacturer with id: " + id, e);
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        String query = "SELECT * FROM manufacturers WHERE deleted = FALSE;";
+        List<Manufacturer> manufacturerList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                manufacturerList.add(createManufacturer(resultSet));
+            }
+            resultSet.close();
+        } catch (SQLException e) {
+            throw new DataProcessingException("No manufacturers found", e);
+        }
+        return manufacturerList;
+    }
+
+    private Manufacturer createManufacturer(ResultSet resultSet) throws SQLException {
+        try {
+            long manufacturerId = resultSet.getObject("manufacturer_id", Long.class);
+            String manufacturerCountry = resultSet.getObject("manufacturer_country", String.class);
+            String manufacturerName = resultSet.getObject("manufacturer_name", String.class);
+            Manufacturer manufacturer = new Manufacturer(manufacturerName, manufacturerCountry);
+            manufacturer.setId(manufacturerId);
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can not parse data"
+                    + " from manufacturer with id: "
+                    + resultSet.getObject("manufacturer_id", Long.class), e);
+        }
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String query = "UPDATE manufacturers"
+                + " SET manufacturer_name = ?, manufacturer_country = ?"
+                + " WHERE manufacturer_id = ?"
+                + " AND deleted = FALSE;";
+
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection
+                        .prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
```

Comment:
```suggestion
                        .prepareStatement(query)) {
```
we don't need to return generated keys here

Reply:
Removed.